rust/agama-server/src/software/web.rs

```diff
@@ -129,6 +132,8 @@ pub async fn software_service(dbus: zbus::Connection) -> Result<Router, ServiceE
         .route("/probe", post(probe))
         .merge(status_router)
         .merge(progress_router)
+        .nest("/issues/product", product_issues)
+        .nest("/issues/software", software_issues)
```

Comment:
to be honest I am not sure if I expect subtree under issue or other way `/software/issues`. Probably later makes more sense to me. Or having general `/issues` call. Would it be possible?